pulp_rpm/app/viewsets.py

```diff
@@ -159,10 +159,10 @@ class UpdateRecordFilter(ContentFilter):
     class Meta:
         model = UpdateRecord
         fields = {
-            'id': ['exact', 'in'],
-            'status': ['exact', 'in'],
-            'severity': ['exact', 'in'],
-            'type': ['exact', 'in'],
+            'id': ['exact', 'in', 'ne'],
```

Comment:
Random rant: I am not convinced, you should ever do a `ne` lookup on id...

Second thought: A 'not in' like filter could be interesting for 'status' and 'severity'.